src/preact.d.ts

```diff
@@ -361,390 +426,413 @@ declare namespace JSX {
-		h2:HTMLAttributes;
-		h3:HTMLAttributes;
-		h4:HTMLAttributes;
-		h5:HTMLAttributes;
-		h6:HTMLAttributes;
-		head:HTMLAttributes;
-		header:HTMLAttributes;
-		hr:HTMLAttributes;
-		html:HTMLAttributes;
-		i:HTMLAttributes;
-		iframe:HTMLAttributes;
-		img:HTMLAttributes;
-		input:HTMLAttributes;
-		ins:HTMLAttributes;
-		kbd:HTMLAttributes;
-		keygen:HTMLAttributes;
-		label:HTMLAttributes;
-		legend:HTMLAttributes;
-		li:HTMLAttributes;
-		link:HTMLAttributes;
-		main:HTMLAttributes;
-		map:HTMLAttributes;
-		mark:HTMLAttributes;
-		menu:HTMLAttributes;
-		menuitem:HTMLAttributes;
-		meta:HTMLAttributes;
-		meter:HTMLAttributes;
-		nav:HTMLAttributes;
-		noscript:HTMLAttributes;
-		object:HTMLAttributes;
-		ol:HTMLAttributes;
-		optgroup:HTMLAttributes;
-		option:HTMLAttributes;
-		output:HTMLAttributes;
-		p:HTMLAttributes;
-		param:HTMLAttributes;
-		picture:HTMLAttributes;
-		pre:HTMLAttributes;
-		progress:HTMLAttributes;
-		q:HTMLAttributes;
-		rp:HTMLAttributes;
-		rt:HTMLAttributes;
-		ruby:HTMLAttributes;
-		s:HTMLAttributes;
-		samp:HTMLAttributes;
-		script:HTMLAttributes;
-		section:HTMLAttributes;
-		select:HTMLAttributes;
-		slot:HTMLAttributes;
-		small:HTMLAttributes;
-		source:HTMLAttributes;
-		span:HTMLAttributes;
-		strong:HTMLAttributes;
-		style:HTMLAttributes;
-		sub:HTMLAttributes;
-		summary:HTMLAttributes;
-		sup:HTMLAttributes;
-		table:HTMLAttributes;
-		tbody:HTMLAttributes;
-		td:HTMLAttributes;
-		textarea:HTMLAttributes;
-		tfoot:HTMLAttributes;
-		th:HTMLAttributes;
-		thead:HTMLAttributes;
-		time:HTMLAttributes;
-		title:HTMLAttributes;
-		tr:HTMLAttributes;
-		track:HTMLAttributes;
-		u:HTMLAttributes;
-		ul:HTMLAttributes;
-		"var":HTMLAttributes;
-		video:HTMLAttributes;
-		wbr:HTMLAttributes;
-
-		//SVG
-		svg:SVGAttributes;
-		animate:SVGAttributes;
-		circle:SVGAttributes;
-		clipPath:SVGAttributes;
-		defs:SVGAttributes;
-		ellipse:SVGAttributes;
-		feBlend:SVGAttributes;
-		feColorMatrix:SVGAttributes;
-		feComponentTransfer:SVGAttributes;
-		feComposite:SVGAttributes;
-		feConvolveMatrix:SVGAttributes;
-		feDiffuseLighting:SVGAttributes;
-		feDisplacementMap:SVGAttributes;
-		feFlood:SVGAttributes;
-		feGaussianBlur:SVGAttributes;
-		feImage:SVGAttributes;
-		feMerge:SVGAttributes;
-		feMergeNode:SVGAttributes;
-		feMorphology:SVGAttributes;
-		feOffset:SVGAttributes;
-		feSpecularLighting:SVGAttributes;
-		feTile:SVGAttributes;
-		feTurbulence:SVGAttributes;
-		filter:SVGAttributes;
-		foreignObject:SVGAttributes;
-		g:SVGAttributes;
-		image:SVGAttributes;
-		line:SVGAttributes;
-		linearGradient:SVGAttributes;
-		marker:SVGAttributes;
-		mask:SVGAttributes;
-		path:SVGAttributes;
-		pattern:SVGAttributes;
-		polygon:SVGAttributes;
-		polyline:SVGAttributes;
-		radialGradient:SVGAttributes;
-		rect:SVGAttributes;
-		stop:SVGAttributes;
-		symbol:SVGAttributes;
-		text:SVGAttributes;
-		tspan:SVGAttributes;
-		use:SVGAttributes;
+}
+
+declare global {
+
+	// augment global Function
```

Comment:
I suppose this is mostly harmless, but it still feels a little dirty to me. Is it really necessary now that these properties are already on `SFC`?